main/common/IO/json_reader.py

```diff
@@ -0,0 +1,27 @@
+'''
+JSON-to-string file reader.
+Input is a .json file, output is a python string that
+is passes on to the deserializer in observation_ticket or filter_wheel.
+'''
+import json
+from main.common.datatype.observation_ticket import ObservationTicket
+from main.common.datatype.filter_wheel import FilterWheel
+
+class Reader():
+    
+    def __init__(self, path=None):
+        self.path = path
+        
+        with open(self.path, 'r') as file:
+            self.__dict__ = json.load(file)
+        
+        self.str = json.dumps(self.__dict__)
+        
+        self._pass_to_deserializer()
+    
+    def _pass_to_deserializer(self):
+        if list(self.__dict__.keys())[0] == 'name':
```

Comment:
There is a better way we can handle this.

Reply:
Then you get rid of the if else statements and just say 

if object in OBJECTS:
   self.ticket = OBJECTS[object].deserialized(self.str)

That way if you ever create a new object you want to have a JSON config for just have the dictionary point to the object and this will point to the deserializer.

Reply:
Ok, I'll start on that and maybe move that part into a separate object reader class like you mentioned in your e-mail.